docs/psa/accel/index.md

```diff
@@ -0,0 +1,8 @@
+# PSA cryptographic accelerator interface
+
+The accelerator interface lets you drivers for cryptographic accelerators into an implementation of the [PSA Cryptography API](../#application-programming-interface). Cryptographic accelerators perform cryptographic operations with keys in clear text.
```

Comment:
```diff
- lets you drivers
+ lets you write drivers
```

Reply:
```diff
- into an implementation of the
+ that interfaces with an implementation of the
```

Although I don't like saying "interface" twice in the sentence, the sentence as written doesn't make much sense to me.

Reply:
I think I started to write “lets you plug … into …” then fiddled with the sentence quite a bit and it's still not quite right. Suggestions welcome. Maybe split this into two sentences?

Reply:
Yes, plug would make more sense. It wasn't the first word that came to mind when I read the first part of the sentence and tried to fill in the gap. Rewrite would be good.